packages/fluentui/react-builder/src/config.tsx

```diff
@@ -538,6 +548,85 @@ export const renderJSONTreeToJSXElement = (
   });
 };
 
+const packageImportList: Record<string, CodeSandboxImport> = {
+  '@fluentui/react-icons-northstar': {
+    version: projectPackageJson.version,
+    module: FUIIcons,
+    required: false,
+  },
+  '@fluentui/react-northstar': {
+    version: projectPackageJson.version,
+    module: FUI,
+    required: true,
```

Comment:
Is there a reason to have `packageImports` & `packageImportsList` separately?

Reply:
Yes, the idea is that `packageImports` are required imports and then based on the components in tree we decide which imports from `packageImportsList` we will add, for example if there are no icons we don't need to add '@fluentui/react-icons-northstar'